cmd/sonobuoy/app/status.go

```diff
@@ -75,9 +74,23 @@ func getStatus(cmd *cobra.Command, args []string) {
 	for _, pluginStatus := range status.Plugins {
 		fmt.Fprintf(tw, "%s\t%s\t%s\n", pluginStatus.Plugin, pluginStatus.Node, pluginStatus.Status)
 	}
-	fmt.Fprintf(tw, "\t\t%s\n", strings.ToUpper(status.Status))
+
 	if err := tw.Flush(); err != nil {
 		errlog.LogError(errors.Wrap(err, "couldn't write status out"))
 		os.Exit(1)
 	}
+	fmt.Printf("\n%s\n", statusAsString(status.Status))
+}
+
+func statusAsString(str string) string {
```

Comment:
Why isn't this on the Status struct? Could be the String() string method even

Reply:
The status struct has a bunch of information on it, not just the overall status. So having it as String() doesn't make much sense I think? But I could move it there for sure.

Reply:
I think you're right, status is the status of a plugin not the overall status. Maybe this could be named `overallStatusMessage` or something like that? I think it's the `AsString` bit that I'm not a fan of since it is redundant with the function signature.

Reply:
how about "humanReadableStatus"?